acquia_lift_profiles/js/acquia_lift_profiles.debug.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Defines JavaScript functions needed by the Acquia Lift debugger in
+ * order to provide customer profile information.
+ */
+
+(function ($, Drupal) {
+
+  Drupal.acquiaLiftProfilesDebug = (function() {
+    var debugPrefix = "acquiaLift::debug::"
+    var curSegmentCapture = {};
+    var curSegments = [];
+    var curSegmentsOverride;
+    var curIdentities = [];
+    var message;
+    var code;
+    $(document).on("segmentsUpdated", function( event, data, capture ) {
+      curSegmentCapture = capture;
+      if(data){
+        curSegments = data["segments"];
+        curSegmentsOverride =  JSON.parse(window.sessionStorage.getItem(debugPrefix + "overrideSegments"));
```

Comment:
I believe that this direct reference to sessionStorage will throw an error if local storage is disabled in the browser.  We have the whole PersonalizeStorage JS library - it should be used in order to standardize the calls to and from local/session storage whenever possible.